tests/models/translationproject.py

```diff
@@ -296,8 +300,16 @@ def test_tp_tool_update(po_directory, tp0, templates):
     tp0.stores.first().delete()
     tp0.stores.first().units.first().delete()
     unit = tp0.stores.first().units.first()
+    unit.source = multistring(["NEW TARGET", "NEW TARGETS"])
     unit.target = "NEW TARGET"
+    unit.context = "something-else"
     unit.save()
+    newunit = unit.__class__()
+    newunit.source = multistring(["OTHER NEW TARGET", "OTHER NEW TARGETS"])
+    newunit.target = "OTHER NEW TARGET"
```

Comment:
perhaps - although this is not something ever checked tested - the issue here was that the proxy class didnt have `hasplural` so would have failed when adding units to a store